test/src/test/java/aima/test/unit/probability/DoubleTypeTest.java

```diff
@@ -8,81 +8,82 @@
 import org.junit.Test;
 
 /**
- * ProbabilityTest to check the various functions of the DoubleProbabilityNumber class
+ * ProbabilityTest to check the various functions of the DoubleProbabilityNumber
+ * class
  */
 public class DoubleTypeTest {
```

Comment:
Shouldn't this be generalized to a ProbabilityNumberTest and test all the different types of probability numbers using JUnits parameterized tests methods?